src/reflect/scala/reflect/internal/tpe/TypeConstraints.scala

```diff
@@ -240,26 +241,20 @@ private[internal] trait TypeConstraints {
           if (up) {
             if (boundSym != AnyClass)
               tvar.addHiBound(bound.instantiateTypeParams(tparams, tvars))
-
-            // Try to derive more constraints for `tvar` (and `tparam`) from its symmetric occurrences in the bounds of other tparams.
-            // `tparam` is the lower bound of `tvarOther`. Flip that, and add `tvarOther` as an upper bound for `tvar`.
-            // Use =:=, so that we equate eta-expanded type constructors (polytypes) and the equivalent no-arg typeref.
-            tvars.foreach { tvarOther =>
-              val tparamOther = tvarOther.origin.typeSymbol
-              if ((tparamOther ne tparam) && tparamOther.info.lowerBound =:= tparamTycon)
-                tvar.addHiBound(tvarOther)
-            }
           } else {
             if (boundSym != tparam && boundSym != NothingClass)
               tvar.addLoBound(bound.instantiateTypeParams(tparams, tvars))
+          }
 
-            // Try to derive more constraints for `tvar` (and `tparam`) from its symmetric occurrences in the bounds of other tparams.
-            // `tparam` is the upper bound of `tvarOther`. Flip that, and add `tvarOther` as an lower bound for `tvar`.
-            // Use =:=, so that we equate eta-expanded type constructors (polytypes) and the equivalent no-arg typeref.
-            tvars.foreach { tvarOther =>
-              val tparamOther = tvarOther.origin.typeSymbol
-              if ((tparamOther ne tparam) && tparamOther.info.upperBound =:= tparamTycon)
-                tvar.addLoBound(tvarOther)
+          // Try to derive more constraints for `tvar` (and `tparam`) from its symmetric occurrences in the bounds of other tparams.
+          // `tparam` is the lower bound of `tvarOther`. Flip that, and add `tvarOther` as an upper bound for `tvar`.
+          // Use =:=, so that we equate eta-expanded type constructors (polytypes) and the equivalent no-arg typeref.
```

Comment:
Is this comment out of date? I don't see an `=:=` below anymore.

Reply:
Yes, I’ll drop it